pandora-common/src/assets/graphics/graphics.ts

```diff
@@ -207,6 +210,7 @@ export enum LayerSide {
 
 export const LayerImageSettingSchema = z.object({
 	image: z.string(),
+	uvPose: z.record(BoneNameSchema, z.number()).optional(),
```

Comment:
it is not quite clear for me what this will be used for, I did not see it used for soles either

maybe we should provide a small description for it

Reply:
if you add it later when we have editor that also fine

Reply:
This is an experiment that generalizes the `scaling` option, but it is unfinished and doesn't produce the best results so far - mostly on the asset side of things. I will add comments explaining that.